### PR TITLE
Added ability to create items directly on a host to the zbx_item ansible module

### PR DESCRIPTION
**Pull request by twiest**

### CODE REVIEW

File: ansible/roles/lib_zabbix/library/zbx_item.py [hunk 1]
```diff
@@ -198,21 +218,58 @@ def main():
...
+                                   'get',
+                                   {'search': {'key_': module.params['key']},
+                                    'selectApplications': 'applicationid',
+                                    'templateids': template_ids,
+                                   })
+
+    if module.params['hostname']:
+        host_ids = get_host_ids(zapi, module.params['hostname'])
+
+        # Fail if a template was not found matching the name
```
Comment (joelddiaz) [inline ansible/roles/lib_zabbix/library/zbx_item.py]: misleading comment

**Comment by joelddiaz** [comment]
:+1:

**Comment by joelddiaz** [comment]
:+1:

**Comment by kwoodson** [comment]
:+1:

**Comment by twiest** [comment]
Merging during freeze because I got 2  +1s.

**Pull request merged**